### DISCUSSION TITLE
Play support for SLF4J 2.0.0?

### DISCUSSION THREAD

**Discussion by jfsoul** [playframework/playframework#11399] [Q&A] [answered]
SLF4J 2.0.0 was [released](https://www.slf4j.org/news.html) recently, and the ecosystem has begun to update (e.g. [logback 1.3/1.4](https://logback.qos.ch/news.html)). 

SLF4J 2.0.0 [has changed the mechanism for finding its logging backend](https://www.slf4j.org/faq.html#changesInVersion200), which appears to be incompatible with Play 2.8. I made some unsuccessful attempts to override the `LoggerConfigurator`, but the `StaticLoggerBinder` was still expected on the classpath (others may have more success!)

Apart from the new features, I'm not sure of these frameworks' policies on backporting fixes for security vulnerabilities (seems likely they would for some time though.) Therefore I'm wondering when this update would be sensible/feasible to include in Play - specifically whether it is one for Play 2.9?

**Comment by ihostage** [comment]
I agree that support SLF4J 2.0 is important for Play. And if it won't be a part of Play `2.9.0`, it should be a part of one of the next patches `2.9.x`.

**Comment by mkurz** [comment]
There is an issue for this now: #11400

**Comment by mkurz** [comment, marked as answer]
We do not automatically upgrade SLF4J to v2 anymore in Play 2.8, however meanwhile Play 2.9 and 3.0 have been released which upgraded that dependency. I recommend upgrading: https://twitter.com/playframework/status/1721576472047911007
